src/syrupy/__init__.py

```diff
@@ -65,7 +65,7 @@ def pytest_sessionstart(session: Any) -> None:
         update_snapshots=config.option.update_snapshots,
         base_dir=config.rootdir,
         is_providing_paths=any(
-            not arg.startswith("-") and glob.glob(arg)
+            not arg.startswith("-") and glob.glob(arg.split("::")[0])
```

Comment:
Support `pytest test_mod.py::TestClass::test_method`
https://docs.pytest.org/en/latest/usage.html

Reply:
hmm.. we should also check if we support the pyargs syntax: `--pyargs <module>`